pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -439,17 +459,45 @@ func registerClusterToCluster(r registry.Registry) {
 					return lv.pollLatency(ctx, setup.dst.db, ingestionJobID, time.Second, workloadDoneCh)
 				})
 
-				t.Status("waiting for replication stream to finish ingesting initial scan")
+				t.L().Printf("waiting for replication stream to finish ingesting initial scan")
 				waitForHighWatermark(t, setup.dst.db, ingestionJobID, sp.timeout/2)
 
-				t.Status("waiting for src cluster workload to complete")
-				m.Wait()
+				t.Status(fmt.Sprintf(`initial scan complete. run workload and repl. stream for another %d minutes`,
+					int(sp.additionalDuration.Minutes())))
```

Comment:
(Here and in another places): `time.Duration` already implements a decent string representation, so you could just use `%s` and pass the duration object directly.

Reply:
TIL. Thanks!